### DISCUSSION TITLE
CORS error: No 'Access-Control-Allow-Origin' header is present

### DISCUSSION THREAD

**Discussion by sl00k** [fastapi/fastapi#6292] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from starlette.middleware.cors import CORSMiddleware

app = FastAPI()
app.add_middleware(
    CORSMiddleware,
    allow_origins=[
        "http://localhost:3000/bar",
    ],
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)
@app.get("/get-data")
def get_data():
    x = {"data": ["test"]}
    return (x)
```

### Description

WEhn I call the IP in the browser it works fine.

In my React script I get an error
![image](https://user-images.githubusercontent.com/33859801/186944112-4257bb1f-1e37-4908-ada1-f0550c32ccc2.png)

I did add the url to allow origins already

### Operating System

Windows

### FastAPI Version

0.80.0

### Python Version

3.9.5

**Comment by JarroVGIT** [comment, marked as answer]
Change `allow_origins=["http://localhost:3000/bar"]` to `allow_origins=["http://localhost:3000"]`

**Comment by sl00k** [comment]
thanks, that was one of the problems.
super stupid. sry

**Comment by JarroVGIT** [comment]
Don’t apologize, we’re all here to help ea cv other out and none of us never had a facepalm moment ourselves 😁 if it solved your issue, please close it, thanks!
